actionpack/test/controller/content_type_test.rb

```diff
@@ -92,14 +92,14 @@ def test_content_type_from_render
   # :ported:
   def test_charset_from_body
     get :render_charset_from_body
-    assert_equal Mime::HTML, @response.content_type
+    assert_equal Mime::TEXT, @response.content_type
     assert_equal "utf-16", @response.charset
   end
 
   # :ported:
   def test_nil_charset_from_body
     get :render_nil_charset_from_body
-    assert_equal Mime::HTML, @response.content_type
+    assert_equal Mime::TEXT, @response.content_type
     assert_equal "utf-8", @response.charset, @response.headers.inspect
   end
 
```

Comment:
It does, actually.

And that's because I uses `TextRenderer` for it. Originally, this `body` format would unset the `Content-Type` header, so I didn't care what format would the renderer return. However, in #14238 we decide that we should not mutate the header hash, and I totally forgot about that.

So, we shipped Rails 4.1 and 4.2 with `render :body` that would return `text/plain` response, instead of `text/html` response like `render :text`.

If you think that's the bug and we should _fix_ it (by making `render :body` actually returns `text/html` by default) I'm all ears.